### PR TITLE
[SCB-1716] Fix high cpu load when there are too many instances

### PR DESCRIPTION
**Pull request by AngLi2**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java [hunk 1]
```diff
@@ -31,9 +31,9 @@
   //// 2.1 configuration items
   public static final String PROP_ROOT = "servicecomb.loadbalance.";
 
-  public static final String RPOP_SERVER_EXPIRED_IN_SECONDS = "servicecomb.loadbalance.stats.serverExpiredInSeconds";
+  public static final String PROP_SERVER_EXPIRED_IN_SECONDS = "servicecomb.loadbalance.stats.serverExpiredInSeconds";
 
-  public static final String RPOP_TIMER_INTERVAL_IN_MINIS = "servicecomb.loadbalance.stats.timerIntervalInMilis";
+  public static final String PROP_TIMER_INTERVAL_IN_MILLIS = "servicecomb.loadbalance.stats.timerIntervalInMilis";
```
Comment (liubao68) [inline handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: change to timerIntervalInMillis

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java [hunk 2]
```diff
@@ -122,10 +122,14 @@ public boolean isRetryEnabled(String microservice) {
   }
 
   public int getRetryOnNext(String microservice) {
+    return getRetry(microservice, PROP_RETRY_ONNEXT);
+  }
+
+  private int getRetry(String microservice, String propRetry) {
```
Comment (liubao68) [inline handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: getRetry can change to a better name

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/ServiceCombLoadBalancerStats.java [hunk 3]
```diff
@@ -104,6 +106,10 @@ public ServiceCombServerStats getServiceCombServerStats(ServiceCombServer server
   }
 
   public ServiceCombServer getServiceCombServer(MicroserviceInstance instance) {
+    return serviceCombServers.get(instance.getInstanceId());
+  }
+
+  public ServiceCombServer getServiceCombServerOld(MicroserviceInstance instance) {
```
Comment (liubao68) [inline handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/ServiceCombLoadBalancerStats.java]: getServiceCombServerOld is never used

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java [hunk 4]
```diff
@@ -29,25 +29,25 @@
...
 public final class Configuration {
   //// 2.1 configuration items
-  public static final String PROP_ROOT = "servicecomb.loadbalance.";
+  public static final String ROOT = "servicecomb.loadbalance.";
 
-  public static final String PROP_SERVER_EXPIRED_IN_SECONDS = "servicecomb.loadbalance.stats.serverExpiredInSeconds";
+  public static final String SERVER_EXPIRED_IN_SECONDS = "servicecomb.loadbalance.stats.serverExpiredInSeconds";
 
-  public static final String PROP_TIMER_INTERVAL_IN_MILLIS = "servicecomb.loadbalance.stats.timerIntervalInMilis";
+  public static final String TIMER_INTERVAL_IN_MILLIS = "servicecomb.loadbalance.stats.timerIntervalInMilis";
```
Comment (liubao68) [inline handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: timerIntervalInMilis not changed

File: foundations/foundation-config/src/main/resources/mapping.yaml [hunk 5]
```diff
@@ -16,3 +16,6 @@
 ## ---------------------------------------------------------------------------
 
 SERVICECOMB_ENV: service_description.environment
+
+servicecomb.loadbalance.stats.timerIntervalInMillis:
+  - servicecomb.loadbalance.stats.timerIntervalInMilis
```
[the remark this answers is not in the document]
Comment (AngLi2) [reply foundations/foundation-config/src/main/resources/mapping.yaml]: done, UTs added. PTAL

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/ServiceCombLoadBalancerStats.java (hunk 3 above)
Comment (AngLi2) [reply handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/ServiceCombLoadBalancerStats.java]: deleted.

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java (hunk 2 above)
Comment (AngLi2) [reply handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: Name changing takes com.netflix.client.DefaultLoadBalanceRetryHandler naming as reference.

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java (hunk 1 above)
Comment (AngLi2) [reply handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: done

File: handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java (hunk 4 above)
Comment (AngLi2) [reply handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java]: done

**Pull request merged**